### PR TITLE
init frontend

### PR DESCRIPTION
**Pull request by queq1890**
# What
- #3 
- basic frontend settings to start developing 

## Tasks : 
- [x] clone react-boilerplate
- [x] create `client` folder and move frontend related files
- [x] delete unused components

### 🚨 Important : I used `yarn` as a package manager 🚨
Because I believe `yarn` is better than `npm` at these 2 points. If there is any problem with `yarn`, 
tell me in this pull request. 🙏 
- package installation is much faster with yarn
- `yarn.lock` allows us to keep node_modules' version specific one

### CODE REVIEW

**Comment by queq1890** [comment]
TODO : create `client` folder in root and move all frontend related stuff into `client`

**Comment by queq1890** [comment]
This pull request is too big because of `react-boilerplate`.

Maybe it's better to check codes from branch.
https://github.com/emily0604/reporting_system/tree/init_frontend

File: client/app/components/LoadingIndicator/tests/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  left: 0;
+  top: 0;
+  -webkit-transform: rotate(30deg);
+  -ms-transform: rotate(30deg);
+  -webkit-transform: rotate(30deg);
+  -ms-transform: rotate(30deg);
+  transform: rotate(30deg);
```
Comment (nvtcp9x) [inline client/app/components/LoadingIndicator/tests/__snapshots__/index.test.js.snap]: You should export this method from a helper.js file, like that:
export const rotate = (rotateValue = '0deg') => `
  -webkit-transform: rotate(rotateValue);
  -ms-transform: rotate(rotateValue);
  -webkit-transform: rotate(rotateValue);
  -ms-transform: rotate(rotateValue);
  transform: rotate(rotateValue);
`
then use it when needed:
.c2 {
  left: 0;
  top: 0;
  ${rotate(30deg)};
}

or if you need to check some conditions before use it:
${({ needToRotate }) => rotate(30deg)}

File: client/app/translations/de.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "boilerplate.components.Footer.author.message":
```
Comment (nvtcp9x) [inline client/app/translations/de.json]: I think you should clear all of these message

File: client/app/translations/en.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "boilerplate.components.Footer.author.message": "Made with love by {author}.",
```
Comment (nvtcp9x) [inline client/app/translations/en.json]: Same here

**Pull request merged**